pages/wiki/useful-commands.md

```diff
@@ -76,11 +76,23 @@ mcetool -M disabled # Disables the tap to unlock functionality
 
 ---
 
-Systemd is the init system used by AsteroidOS. It is controlled by a command named `systemctl`. This command can be used to restart the user session or the entire system into different mode.
+Systemd is the init system used by AsteroidOS. It is controlled by a command named `systemctl`. This command can be used to restart the user session or the entire system into a different mode.
 
 ```
 systemctl restart user@1000 # Restarts the ceres session (including the entire UI)
 reboot # Simply reboot your watch
 systemctl --force reboot bootloader # Restarts the watch in bootloader mode
 systemctl --force reboot recovery # Restarts the watch in recovery mode
 ```
+
+# Timezone, date & time
+
+---
+
+Although we offer [synchronization clients](https://asteroidos.org/wiki/synchronization-clients/) for different platforms, it might still be useful to set the time zone or synchronise the date and time using standard Linux tools. The Linux folder `/usr/share/zoneinfo/` contains the nameing scheme for your local time zone in <continent>/<zone> format.
```

Comment:
`nameing` → `naming`

`time zone in <continent>/<zone> format.` results in `time zone in / format.` since the `<words>` are interpreted as HTML tags. This needs either backticks or code tags to make it displayed correctly.

Reply:
Good spot :D
Sorry. 
Also what i see only now. What about `Timezone` vs `time zone` in title and text? Deepl says `Time zone` now when i check.

Reply:
I would be consistent and use `time zone`. In a headline you can use a consistent title case instead.